spec/spec/fx.js

```diff
@@ -40,12 +41,26 @@ describe('FX', function() {
     it('converts a timestamp to a progress', function() {
       expect(fx.timeToPos(fx.situation.start+fx.situation.duration/2)).toBe(0.5)
     })
+
+    it('should take speed into consideration', function() {
+      fx.speed(4)
+      expect(fx.timeToPos(fx.situation.start+fx.situation.duration/4/2)).toBe(0.5)
+      fx.speed(0.5)
+      expect(fx.timeToPos(fx.situation.start+fx.situation.duration/0.5/4)).toBe(0.25)
+    })
   })
 
   describe('posToTime()', function() {
     it('converts a progress to a timestamp', function() {
       expect(fx.posToTime(0.5)).toBe(fx.situation.start+fx.situation.duration/2)
     })
+
+    it('should take speed into consideration', function() {
+      fx.speed(4)
+      expect(fx.posToTime(0.5)).toBe(fx.situation.start+fx.situation.duration/4/2)
```

Comment:
`fx.situation.start+fx.situation.duration/4/2` is not very clear. Can you use variables to better communicate what you're testing here?
The same in the similar test cases. The tests need to be understandable to someone who hasn't written the implementation.